### PR TITLE
Added db seeding using Faker

### PR DESCRIPTION
**Pull request by GRSDY**
Use `rails db:seed` to run.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,75 @@
...
+    for i in (1..2)
+      InstrumentCategory.new do |instcat|
+        instrument_categories_created += 1
+        instcat.category = Faker::Music.instrument
+        instcat.created_at = Faker::Date.backward(days: 14)
+        instcat.updated_at = Faker::Date.backward(days: 14)
+
+        for i in (1..2)
+          instrument = Instrument.new do |inst|
+            instruments_created += 1
```
Comment (isaquito1993) [inline db/seeds.rb:34]: we also need some names, description, price and type for the instruments

**my suggestions**

Name:
Faker::Lorem.sentence(word_count: 3) #=> "Quaerat quam unde."

Description:
Faker::Lorem.paragraph #=> "Recusandae minima consequatur. Expedita sequi blanditiis. Ut fuga et."

Price:
Faker::Number.between(from: 10, to: 50)

type:
Faker::Music.instrument #=> "Ukelele"

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,75 @@
...
+            inst.instrument_category = instcat
+            inst.user = u
+            inst.created_at = Faker::Date.backward(days: 14)
+            inst.updated_at = Faker::Date.backward(days: 14)
+
+            for i in (1..2)
+              review = Review.new do |rev|
+                reviews_created += 1
+                rev.user = u
```
Comment (isaquito1993) [inline db/seeds.rb:43]: here id also suggest to add some more info for rate and content

Rate:
Faker::Number.between(from: 1, to: 5)

Content:
Faker::Lorem.paragraph #=> "Recusandae minima consequatur. Expedita sequi blanditiis. Ut fuga et."

**Review by isaquito1993 — COMMENTED** [review]
have left some comments in the file

i think we need a bit more fake data, so we can display it.

File: db/seeds.rb (hunk 2 above)
Comment (GRSDY) [reply db/seeds.rb:43]: Rate and content are already added down on line 47 and 48. Content uses restaurant reviews even

File: db/seeds.rb (hunk 1 above)
Comment (GRSDY) [reply db/seeds.rb:34]: Added all of those :)

File: db/seeds.rb (hunk 2 above)
Comment (isaquito1993) [reply db/seeds.rb:47]: yessss. you are very write. nice one.

**Review by isaquito1993 — APPROVED** [review]
thanks for adding the changes. all good from my side

**Pull request merged**